cmd/print.go

```diff
@@ -219,6 +254,22 @@ func (pr *PrintRunner) initializeNamespaceFilter() error {
 	return nil
 }
 
+// 1. When --output-file receives no argument, initializeOutputFile will do nothing.
+// 2. When --output-file receives an argument, initializeOutputFile will validate whether the value is correct.
+func (pr *PrintRunner) initializeOutputFile() {
+	if pr.outputFile == "" {
+		return
+	}
+	if !strings.HasSuffix(pr.outputFile, ".json") && !strings.HasSuffix(pr.outputFile, ".yaml") {
+		fmt.Println("Warning: output file should be *.json or *.yaml")
+	} else if strings.HasSuffix(pr.outputFile, ".json") && pr.outputFormat != "json" {
+		fmt.Println("Warning: output format is yaml, but output file is *.json")
+	} else if strings.HasSuffix(pr.outputFile, ".yaml") && pr.outputFormat != "yaml" {
+		fmt.Printf("Warning: output format is json, but output file is *.yaml")
+	}
+	return
+}
```

Comment:
I do not think we need this logic at all, we should use the file provided by the user regardless the extension in my opinion, without logging any specific message